src/modules/Dataset/Editions.vue

```diff
@@ -68,10 +68,9 @@ export default {
         this.loading = true
 
         const { data: editions } = await this.$axios.request({
```

Comment:
Tror du kun trenger:
```suggestion
        const { data: editions } = await this.$axios.get(`/api/dataplatform/metadata/datasets/${this.datasetId}/versions/1/editions`)
```

Reply:
@jonmorten Da får jeg "Uncaught (in promise) TypeError: relativeURL.replace is not a function"

Reply:
@jonmorten Glem det! ;)